### PR TITLE
Add EIP-6466: SSZ receipts root

### PR DESCRIPTION
**Pull request by etan-status**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by SamWilsn** [comment]
@eth-bot rerun

File: EIPS/eip-6466.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+eip: 6466
+title: SSZ receipts root
```
Comment (SamWilsn) [inline EIPS/eip-6466.md:3]:
```suggestion
title: SSZ Receipts Root
```

File: EIPS/eip-6466.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-02-08
+requires: 155, 658, 1559, 2718, 2930, 4844, 6404
+---
+
+## Abstract
+
+This EIP defines a migration process of existing Merkle-Patricia Trie (MPT) commitments for receipts to SSZ.
```
Comment (SamWilsn) [inline EIPS/eip-6466.md:16]:
```suggestion
This EIP defines a migration process of existing Merkle-Patricia Trie (MPT) commitments for receipts to Simple Serialize (SSZ).
```

File: EIPS/eip-6466.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+## Motivation
+
+While the consensus `ExecutionPayloadHeader` and the execution block header map to each other conceptually, they are encoded differently. This EIP aims to align the encoding of their fields, taking advantage of the more modern SSZ format. This brings several advantages:
+
+1. **Reducing complexity:** Merkle-Patricia Tries (MPT) are hard to work with. Replacing them with SSZ leaves only the state trie in the legacy MPT format.
+
+2. **Better for smart contracts:** The SSZ format is optimized for production and verification of merkle proofs. It allows proving specific fields of containers and allows chunked processing.
+
+3. **Better for light clients:** Light clients with access to the consensus `ExecutionPayload` no longer need to implement Merkle-Patricia Trie or RLP libraries to work with receipts.
```
Comment (SamWilsn) [inline EIPS/eip-6466.md:26]: This looks similar to the Motivation from EIP-6404. Is that intentional?

**Review by SamWilsn — COMMENTED** [review]
Just in case you aren't aware, you'll have to rebase this on top of `master` to fix the missing file errors.

Otherwise, looking great!

File: EIPS/eip-6466.md (hunk 3 above)
Comment (etan-status) [reply EIPS/eip-6466.md:26]: Same motivation for all three: EIP-6404 for transactions, EIP-6465 for receipts, and EIP-6466 for withdrawals. The three MPTs are independent of each other and don't necessarily be linked together.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**